packages/astro/src/default/layouts/Layout.astro

```diff
@@ -4,19 +4,27 @@ import { joinPaths } from '../utils/url';
 
 interface Props {
   title: string;
+  description?: string;
 }
 
-const { title } = Astro.props;
+const { title, description = 'TutorialKit' } = Astro.props;
 const baseURL = import.meta.env.BASE_URL;
+const logoUrl = joinPaths(baseURL, '/logo.svg');
```

Comment:
We should probably move the logic from `Logo.astro` in an utility function as we support multiple formats. Here the logo has to be ending with `.svg` which might not be the case.

So you want to copy the logic from here:

https://github.com/stackblitz/tutorialkit/blob/c8ce1826aeb79ef878a4e5e7104369ce900eb76d/packages/astro/src/default/components/Logo.astro#L12-L26

To a new file, like `src/default/utils/logo.ts`. You'll want to move the `LOGO_EXTENSIONS` constant as well.

Reply:
Yep done, I've also crafted an utility for the favicon (with only SVG support). And documented BASE_URL in the deployment section of the doc, to explain how to get absolute URLs that are needed for open graph.